Dockerfile

```diff
@@ -220,11 +220,11 @@ ENV XDEBUG_CLIENT_HOST="host.docker.internal"
 
 # Copy Dev Scripts
 COPY docker/*-dev /usr/local/bin/
-RUN chmod +x /usr/local/bin/*-dev; \
+RUN chmod +x /usr/local/bin/*-dev;
 
 # ------------------------------------------------------ PHP -----------------------------------------------------------
 
-    mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini" \
```

Comment:
To avoid an unnecessary `RUN` and a new image layer, we should keep those two commands in the same `RUN` instruction. Removing the extra newline, and trailing slash will fix it without having to have another `RUN`.